### ISSUE TITLE
number grouping separator escaping

### ISSUE THREAD

**Issue by ideadapt** [tinganho/l10ns#67]
When I compile with a `de-CH` locale, l10ns outputs unescaped literals resulting in invalid json. 
```
'__numberSymbols': {
        'latn': {
          'decimal': '.',
          'group': ''' // invalid syntax here, '\'' would be valid
        }
      },
```

**Comment by tinganho** [comment]
Ok I will try to resolve this issue and try to push out a release directly so you can use l10ns.

**Comment by tinganho** [comment]
I just fixed this issue in `l10ns@1.0.3`. Just run `sudo npm install l10ns -g`and it will install the latest version of `l10ns`. Let me know if you have any problem. If it is any critical I can fix it directly.

**Comment by ideadapt** [comment]
@tinganho thx for your immediate action. it works now.
